### PR TITLE
feat(EMS-2182): No PDF - Your business - start page 

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction ✏️ 
This PR adds a start page to the  "your business" section of an application.

## Resolution ✔️ 
- Add route, controller, content strings and template for the root of "your business".
- Update the "your business" task to redirect to the new "start" route.
- Update the "policy and exports - check your answers" redirect to the new route/start page, instead of to "company details".
- Update appropriate E2E tests to click on the "start now" link via new cypress command, `startYourBusinessSection`; Removing the need to import and consume `task` partials.

## Miscellaneous ➕
- Update "your business" route constants in E2E tests to be the same as the UI constants.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js [hunk 1]
```diff
@@ -63,8 +61,9 @@ context('Insurance - Your business - Company details page - As an Exporter I wan
   it('renders core page elements', () => {
     cy.corePageChecks({
       pageTitle: CONTENT_STRINGS.PAGE_TITLE,
-      currentHref: `${ROOT}/${referenceNumber}${COMPANY_DETAILS}`,
-      backLink: `${ROOT}/${referenceNumber}${ALL_SECTIONS}`,
+      currentHref: `${ROOT}/${referenceNumber}${COMPANY_DETAILS_ROOT}`,
+      backLink: `${ROOT}/${referenceNumber}${EXPORTER_BUSINESS_ROOT}`,
+      // backLink: `testing`,
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js:66]: remove?

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/root/your-business-root.spec.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.business;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Your business - Alternative trading address page - I want to input information on an alternative business trading address So that I can provide necessary business information to support my application for Export Insurance', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/root/your-business-root.spec.js:30]: wrong context?

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js]: Whoops, removed

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/root/your-business-root.spec.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/your-business/root/your-business-root.spec.js]: Great spot, updated

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**